thorlcr/activities/indexwrite/thindexwriteslave.cpp

```diff
@@ -241,6 +243,8 @@ class IndexWriteSlaveActivity : public ProcessSlaveActivity, public ILookAheadSt
                     numBranchNodes = builder->getNumBranchNodes();
                     numBlobNodes = builder->getNumBlobNodes();
                     offsetBranches = builder->getOffsetBranches();
+                    branchMemorySize = builder->getBranchMemorySize();
+                    leafMemorySize = builder->getLeafMemorySize();
```

Comment:
not for now, but these stats should be collected/serialized in a more conventional way, whilst the query is running (in gatherActiveStats). Would need IKeyBuilder to expose its stats as CRuntimeStatisticCollection or similar.
I've opened a new JIRA: https://track.hpccsystems.com/browse/HPCC-29880